Classes/core/entity/Attack-down.h

```diff
@@ -0,0 +1,23 @@
+#ifndef TOWERDEFENCE_ATTACK_DOWN_H
+#define TOWERDEFENCE_ATTACK_DOWN_H
+
+#include "entity.h"
+
+namespace towerdefence {
+    namespace core {
+
+        struct Attack_down final : Enemy {
```

Comment:
```suggestion
        struct AttackDown final : Enemy {
```

use CamelCase on types, e.g. `AttackDown`.